lib/OA/Preferences.php

```diff
@@ -608,27 +606,149 @@ function disableStatisticsColumns($aColumns)
+            'default_banner_weight'                         => ['account_type' => OA_ACCOUNT_ADVERTISER,   'default' => 1],
+            'default_campaign_weight'                       => ['account_type' => OA_ACCOUNT_ADVERTISER,   'default' => 1],
+            'warn_email_admin'                              => ['account_type' => OA_ACCOUNT_ADMIN,        'default' => true],
+            'warn_email_admin_impression_limit'             => ['account_type' => OA_ACCOUNT_ADMIN,        'default' => 100],
+            'warn_email_admin_day_limit'                    => ['account_type' => OA_ACCOUNT_ADMIN,        'default' => 1],
+            'campaign_ecpm_enabled'                         => ['account_type' => OA_ACCOUNT_MANAGER,      'default' => false],
+            'contract_ecpm_enabled'                         => ['account_type' => OA_ACCOUNT_MANAGER,      'default' => false],
+            'warn_email_manager'                            => ['account_type' => OA_ACCOUNT_MANAGER,      'default' => true],
+            'warn_email_manager_impression_limit'           => ['account_type' => OA_ACCOUNT_MANAGER,      'default' => 100],
+            'warn_email_manager_day_limit'                  => ['account_type' => OA_ACCOUNT_MANAGER,      'default' => 1],
+            'warn_email_advertiser'                         => ['account_type' => OA_ACCOUNT_ADVERTISER,   'default' => true],
+            'warn_email_advertiser_impression_limit'        => ['account_type' => OA_ACCOUNT_ADVERTISER,   'default' => 100],
+            'warn_email_advertiser_day_limit'               => ['account_type' => OA_ACCOUNT_ADVERTISER,   'default' => 1],
+            'timezone'                                      => ['account_type' => OA_ACCOUNT_MANAGER,      'default' => ''],
+            'tracker_default_status'                        => ['account_type' => OA_ACCOUNT_ADVERTISER,   'default' => MAX_CONNECTION_STATUS_APPROVED],
+            'tracker_default_type'                          => ['account_type' => OA_ACCOUNT_ADVERTISER,   'default' => MAX_CONNECTION_TYPE_SALE],
+            'tracker_link_campaigns'                        => ['account_type' => OA_ACCOUNT_ADVERTISER,   'default' => false],
+            'ui_show_campaign_info'                         => ['account_type' => OA_ACCOUNT_ADVERTISER,   'default' => true],
+            'ui_show_banner_info'                           => ['account_type' => OA_ACCOUNT_ADVERTISER,   'default' => true],
+            'ui_show_campaign_preview'                      => ['account_type' => OA_ACCOUNT_ADVERTISER,   'default' => false],
+            'ui_show_banner_html'                           => ['account_type' => OA_ACCOUNT_ADVERTISER,   'default' => false],
+            'ui_show_banner_preview'                        => ['account_type' => OA_ACCOUNT_ADVERTISER,   'default' => true],
+            'ui_html_wyswyg_enabled'                        => ['account_type' => null,                    'default' => false],
```

Comment:
The fact I've moved public methods above the private ones doesn't allow to clearly see that this line was missing. Hence the bug of the missing entry on new installs.